panel/src/components/Forms/LoginCode.vue

```diff
@@ -37,9 +37,9 @@
 				variant="filled"
 			>
 				{{ $t("login" + (mode === "password-reset" ? ".reset" : "")) }}
-				<template v-if="isLoadingLogin"> … </template>
+				<template v-if="isLoading"> … </template>
```

Comment:
I think it would be nice to have the same submitText method here to keep it in line with the Login.vue component and simplify the template.